hw2/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>HT2</title>
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Fira+Sans" rel="stylesheet">
+</head>
+<body>
+	<div class="container">
+		<header class="header">
+			<div class="header__logo">
+				<img src="pic/logo.png" alt="Fitness">
+			</div>
+			<nav class="menu-top">
+				<a href="#" title="Главная" class="menu-top__item">Главная</a>
+				<a href="#" title="Посмотреть весь асортимент" class="menu-top__item">Товары</a>
+				<a href="#" title="Узнай нашу историю" class="menu-top__item">О_нас</a>
+				<a href="#" title="Мы онлайн 24/7" class="menu-top__item">Контакты</a>
+			</nav>
+		</header>
+		<aside class="main-content clearfix">
```

Comment:
you haven't here any floats. And there are no any columns inside so you won't anyway